lib/matplotlib/tests/test_tickedstroke.py

```diff
@@ -0,0 +1,64 @@
+from matplotlib.testing.decorators import image_comparison
```

Comment:
This should go in `test_patheffects.py`?

Reply:
I can change this if you (or others) feel strongly.  I made it separate because that made sense for early development and testing.

I would probably leave it separate as this also relies on a number of other high-level capabilities to work -- mainly contour and legend.  I.e. it isn't just a low-level test of the patheffect mechanism.